### PR TITLE
Image input

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
finished with the last p0 components.

### CODE REVIEW

File: pages/image/image.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.square-img {
+  width: 150px;
+  height: 150px;
+  border-radius: 5px;
```
Comment (kart-c) [inline pages/image/image.css:4]: Hey Kevin, just a suggestion, the image looks kind of distorted here. You can use `object-fit: cover;`  so the image will fill the height and width of its box.

File: pages/image/image.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+          <p class="small-text">
+            Images make the website more interactive, as they display more context to the user.
+          </p>
+        </article>
+        <article class="section-content underline">
+          <h2>Square Image</h2>
+          <p class="small-text">Square images as the name suggest are squared with a default dimension of 150*150. </p>
+          <div class="example-code">
+            <div class="example-container-flex">
+              <img src="../../assets/image1.jpg" alt="" class="square-img">
```
Comment (kart-c) [inline pages/image/image.html]: you should add some default `alt` attribute

**Review by kart-c — APPROVED** [review]
Added 2 suggestions. Other than that, everything else looks good.

File: pages/image/image.html (hunk 2 above)
Comment (VishalPatil18) [inline pages/image/image.html]: **Fix:** Give value to the _"alt"_ attribute. It helps with accessibility.

**Review by VishalPatil18 — CHANGES_REQUESTED** [review]
Just one small fix. Apart from that everything works well. Great Job Kevin 👍

File: pages/image/image.css (hunk 1 above)
Comment (Kevin-Solomon) [reply pages/image/image.css:4]: Done !!

File: pages/image/image.html (hunk 2 above)
Comment (Kevin-Solomon) [reply pages/image/image.html]: Done !!

Comment (Kevin-Solomon) [reply pages/image/image.html]: Fixed it ,Vishal .

**Review by Aazam10 — APPROVED** [review]
Everything looks good.

**Review by kart-c — APPROVED** [review]
Looks great Kevin 👍

**Review by VishalPatil18 — APPROVED** [review]

**Pull request merged**